apps/developers/docusaurus.config.js

```diff
@@ -0,0 +1,126 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github')
+const darkCodeTheme = require('prism-react-renderer/themes/dracula')
+
```

Comment:
are we still using docusaurus?

Reply:
At first I was thinking of using it for the SDK docs so we could have live code examples, especially for React components having an interactive "connect" button for example.
For now we opted for having the Self.ID docs part of the Ceramic docs so this is less relevant, but I found having this docusaurus setup useful for integration tests.